src/BloomExe/Publish/AudioProcessor.cs

```diff
@@ -87,6 +87,12 @@ internal static string GetAudioFolderPath(string bookFolderPath)
 			return Path.Combine(bookFolderPath, "audio");
 		}
 
+		internal static bool HasAnyAudioFiles(string bookFolderPath)
+		{
+			var audioFolderPath = GetAudioFolderPath(bookFolderPath);
+			return Directory.Exists(audioFolderPath) && Directory.EnumerateFiles(audioFolderPath).Any();
```

Comment:
Some of the other functions in this class only care about specific file extensions.  Do you think this function would want to check only for valid file extensions or to accept any file (which theoretically might not be created by us)?